lib/patterns/teams.json

```diff
@@ -0,0 +1,5 @@
+{
+	"__file": "wp_block",
+	"title": "Teams",
```

Comment:
We'll need to consider translations here eventually.

Reply:
Right! since this is potentially just a temporary location, I'm ok just using php for these for now.

`lib/pattterns/teams.php`